domain.go

```diff
@@ -36,15 +38,92 @@ type DomainUpdateInput struct {
 }
 
 func (s *DomainId) ChildSystems(client *Client, variables *PayloadVariables) (*SystemConnection, error) {
-	return &SystemConnection{}, nil
+	var q struct {
+		Account struct {
+			Domain struct {
+				ChildSystems SystemConnection `graphql:"childSystems(after: $after, first: $first)"`
+			} `graphql:"domain(input: $domain)"`
+		}
+	}
+	if s.Id == "" {
+		return nil, fmt.Errorf("Unable to get Systems, invalid domain id: '%s'", s.Id)
+	}
+	if variables == nil {
+		variables = client.InitialPageVariablesPointer()
+	}
+	(*variables)["domain"] = IdentifierInput{
```

Comment:
I wanted to use NewIdentifier here since DomainId is an Identifier but it wasn't setting the query variable types correctly.  I have a feeling I'm missing something but not sure what.

Reply:
Shouldn't matter though since `s` should always have an ID?

Reply:
This could be 2 things:
1.) `NewIdentifier` returns `*IdentifierInput` and here you are not using the pointer type but the value type
2.) the "ID" you are using in testing doesn't actually base64 decode properly so it thinks its an alias in the `IsID` check here - https://github.com/OpsLevel/opslevel-go/blob/5f92bd8065dd278c7f525f9c6100d0531b68a9ad/scalar.go#L39

Reply:
Went around and fixed this everywhere.